rmgpy/cantherm/statmech.py

```diff
@@ -613,6 +620,12 @@ def applyEnergyCorrections(E0, modelChemistry, atoms, bonds,
                 atomEnergies = {'H':-0.499946213243 + SOC['H'], 'N':-54.588545831900 + SOC['N'], 'O':-75.065995072347 + SOC['O'], 'C':-37.844662139972+ SOC['C']}
             elif modelChemistry == 'ccsd(t)-f12/cc-pcvqz-f12':
                 atomEnergies = {'H':-0.499994558325 + SOC['H'], 'N':-54.589137594139+ SOC['N'], 'O':-75.067412234737+ SOC['O'], 'C':-37.844893820561+ SOC['C']}
+            elif modelChemistry == 'ccsd(t)-f12/cc-pvtz-f12(-pp)':
+                atomEnergies = {'H':-0.499946213243, 'N':-54.53000909621, 'O':-75.004127673424, 'C':-37.789862146471, 'S':-397.675447487865, 'I':-294.81781766}
+            elif modelChemistry == 'ccsd(t)-f12/aug-cc-pvtz(-pp)':
+                atomEnergies = {'H':-0.499844820798 + SOC['H'], 'N':-54.527419359906 + SOC['N'], 'O':-75.000001429806+ SOC['O'], 'C':-37.788504810868+ SOC['C'], 'S':-397.615032078484, 'I':-294.816178186847}
+            elif modelChemistry == 'ccsd(t)/aug-cc-pvtz(-pp)':
+                atomEnergies = {'H':-0.499821176024 + SOC['H'], 'N':-54.417460988701 + SOC['N'], 'O':-74.97498502 + SOC['O'], 'C':-37.729456757559 + SOC['C'], 'S':-397.6242802 + SOC['S'], 'I':-294.8436651}
```

Comment:
If these are fitted, can you add a comment briefly describing the procedure? How come some of them have spin-orbit corrections, while others don't? (e.g. no SOC for sulfur for ccsd(t)-f12/aug-cc-pvtz(-pp))

Reply:
They were all fit to about 5-8 iodine containing molecules. I'll include a more detailed note. I expect the disparity in SOC is whether they were fit with them or not but I'll change them so that the f12 method includes the SOC.